base/install_iocs.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -ex
+
+. /opt/epics/install-functions.sh
+
+#Build OPCUA IOC
+lnls-get-n-unpack -l https://github.com/epics-modules/opcua/releases/download/v${OPCUA_VERSION}/IOC_opcua-${OPCUA_VERSION}_Base-${EPICS_BASE_VERSION}_debian${DEBIAN_VERSION%.*}.tar.gz
+mv binaryOpcuaIoc opcua
+
+install_ioc opcua OPCUA "
+EPICS_BASE
+"
+
+EPICS_HOST_ARCH=`perl ${EPICS_BASE_PATH}/lib/perl/EpicsHostArch.pl`
+ln -s ${EPICS_IOCS_PATH}/opcua/{opcuaIocApp/libopcua.so.0.9,lib/${EPICS_HOST_ARCH}/libopcua.so}
+
+#Build ether_ip IOC
+install_github -i epics-modules ether_ip ETHER_IP $ETHER_IP_VERSION "
+EPICS_BASE
+"
```

Comment:
This comment isn't necessary, and `ether_ip` should be installed in `install_modules.sh`, and its version should be grouped along with the modules installed on that script.

`ether_ip` addition should therefore be a separate commit.

Reply:
> `ether_ip` should be installed in `install_modules.sh`

I like the idea of having it in a separate file if we have multiple IOCs being built in the base image, which distinguishes better support modules and other applications. For instance, musl-based image does not need to install `ether_ip`, because it won't generate a static library to be linked against another application.

Beyond that conceptual division, it is practical to have them installed somewhere else. Pruning can then [simply throw the entire root directory away](https://github.com/cnpem/epics-in-docker/pull/59#issuecomment-2059673694). This could still be scripted inside `install_module.sh` nevertheless.